mobile/lib/src/presentation/pages/settings/settings_page.dart

```diff
@@ -0,0 +1,164 @@
+import 'dart:ui';
+import 'package:enum_to_string/enum_to_string.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:polydodo/src/application/settings/settings_cubit.dart';
+import 'package:polydodo/src/common/constants.dart';
+import 'package:polydodo/src/domain/settings/settings.dart';
+import 'package:polydodo/src/presentation/navigation/navdrawer_widget.dart';
+import 'package:polydodo/src/presentation/widgets/loading_indicator.dart';
+import 'package:settings_ui/settings_ui.dart';
+
+class SettingsPage extends StatefulWidget {
+  SettingsPage({Key key}) : super(key: key);
+
+  @override
+  _SettingsPageState createState() => _SettingsPageState();
+}
+
+class _SettingsPageState extends State<SettingsPage> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(title: Text('Settings')),
+      drawer: NavDrawer(activeTab: NavdrawerTab.Settings),
+      body: BlocBuilder<SettingsCubit, SettingsState>(
+        builder: (context, state) {
+          return state is SettingsLoadSuccess
+              ? SettingsList(
+                  sections: [
+                    SettingsSection(
+                      title: 'Personnal informations',
+                      tiles: [
+                        _buildDatePickerSettingTile(
+                            AGE_KEY, 'In years', Icons.cake, context, state),
+                        _buildPopupListSettingTile(SEX_KEY, Sex.values,
+                            'Your biological sex', Icons.face, state),
+                        _buildOpenTextSettingTile(
+                            SERVER_URL_KEY,
+                            'The url for classification',
+                            Icons.dns,
+                            context,
+                            state),
+                      ],
+                    ),
+                  ],
+                )
+              : LoadingIndicator();
+        },
+      ),
+    );
+  }
+}
+
+SettingsTile _buildDatePickerSettingTile(String title, String substitle,
+    IconData icon, BuildContext context, SettingsState state) {
+  return SettingsTile(
+    title: AGE_KEY,
+    subtitle: 'In years',
+    leading: Icon(Icons.cake),
+    trailing: TextButton(
+        child: Text(
+          (state as SettingsLoadSuccess).settings[AGE_KEY] == null
+              ? 'Not Set'
+              : (state as SettingsLoadSuccess).settings[AGE_KEY].toString(),
+        ),
+        onPressed: () => _showDatePicker(context)),
+  );
```

Comment:
AGE_KEY is too specific. You gave it as a parameter, you should use it.

Reply:
If we create a SETTINGS_KEYS map with an 'AGE' key it would be more clear imo